Tests/NumLib/LocalToGlobalIndexMap.cpp

```diff
@@ -163,7 +163,7 @@ TEST_F(NumLibLocalToGlobalIndexMapTest, DISABLED_MultipleVariablesMultipleCompon
         NumLib::ComponentOrder::BY_COMPONENT);
 
     ASSERT_EQ(30, dof_map->dofSizeWithGhosts());
-    ASSERT_EQ(3u, dof_map->getNumberOfComponents());
+    ASSERT_EQ(3, dof_map->getNumberOfComponents());
     ASSERT_EQ(2u, dof_map->getNumberOfVariables());
```

Comment:
dof_map->getNumberOfVariables() also returns int. Therefore, 2u --> 2.

Reply:
Thanks! I found few more places... Unfortunately I was not able to find/create an automatic substitution.